back-end/api/controllers/transcript_controller.spec.js

```diff
@@ -40,13 +40,13 @@ describe("TranscriptController", () => {
     it("Should correctly post transcript", async() => {
         TranscriptController.setTranscriptInteractor(PostTranscriptInteractor);
         await TranscriptController.postTranscript({body:{payload : {"a" : "a"}}}, mockResponse(), {});
-        expect(PostTranscriptInteractor.formatTranscript).toHaveBeenCalledWith({"a": "a"}, undefined);
+        expect(PostTranscriptInteractor.formatTranscript).toHaveBeenCalledWith({"a": "a"});
     });
 
     it("Should correctly post transcript with override", async() => {
         TranscriptController.setTranscriptInteractor(PostTranscriptInteractor);
         await TranscriptController.postTranscript({body:{payload : {"a" : "a", override: true}}}, mockResponse(), {});
-        expect(PostTranscriptInteractor.formatTranscript).toHaveBeenCalledWith({"a": "a", override: true}, true);
+        expect(PostTranscriptInteractor.formatTranscript).toHaveBeenCalledWith({"a": "a"});
     });
 
```

Comment:
I know you just merged this, but this test is now identical to the one above and somewhat irrelevant. Can you make a super quick PR to delete it (or include that in your next PR)?

Reply:
Yeah I'll do this in my next PR since I'll be working with back-end anyways